### ISSUE TITLE
[Help Wanted] Is there a way to select _all_ notes in your account, not just the currently loaded ones?

### ISSUE THREAD

**Issue by evazorro** [standardnotes/forum#2404]
**Describe the issue**
I have ~2,300 notes in my account. I want to select all of them and delete all but one. I can't figure out how to select _all_ notes, and it will take too long for me to delete them page by page.

**To Reproduce**
1. Click into your notes list.
2. Do Ctrl+A (or Cmd+A on Mac) to select all notes.
3. Right click > Delete.

**Expected behavior**
Expected to see an empty list because you've deleted all the notes. Instead, another older set of notes moves to the top. I think Ctrl+A is only selecting as many notes as are currently loaded based on scroll position. I would like a way to select _all_ notes in my account, not just the current paginated set of notes.

**Screenshots**

**Desktop (please complete the following information):**
 - OS: Windows 11
 - Desktop app
 - Version 3.135.1

**Additional context**
The context is that I have a large number of notes, and I want to delete them all without deleting and recreating my account. But if I have to delete them in batches, based on the speed that was going, it would take me prohibitively long.

**Comment by effieeee** [comment]
Hi @evazorro, unfortunately there isn't a way to do that at the moment, so you really will need to delete them in batches, except for the one note that you wish to keep 😞 Otherwise, the alternative would really involve deleting your account. Specifically, you'll have to export the one note you want to keep, then [delete your current account](https://standardnotes.com/reset) and create a new one using the same email address.

If you perhaps have a subscription tied to your current account, you should receive another email from us upon creating a new account using the same email address. The email will allow you to link your existing subscription with the new account!

**Comment by evazorro** [comment]
Hey @effieeee, thanks for getting back to me! I might go with deleting my account and creating a new one with the same email address. I do have a subscription, so I will look for the email! Thanks again.

**Comment by evazorro** [comment]
Hey Effie, it looks like the delete account function is failing (tried on the web app and Windows desktop app). I went to account > Delete my account > entered my password > I saw a brief message about generating keys, then it just returned me to the account setting screen. I tried clicking Delete my account a couple more times, it looks like the API call is 401ing:

<img width="411" alt="image" src="https://user-images.githubusercontent.com/1493712/213920623-39af4f4e-fb93-466c-9d62-c27974a52389.png">

I can still reload the app (web or desktop) and see all my notes.
